vars/commonPipelineEnvironment.groovy

```diff
@@ -171,43 +173,53 @@ class commonPipelineEnvironment implements Serializable {
         })
     }
 
-    void readFromDisk() {
+    void readFromDisk(script) {
```

Comment:
any reason for adding `script` here. I guess I overlook something here ...

Structure of `commonPipelineEnvironment` allows to call steps directly

Reply:
PR is too old, I don't remember the details reliably. AFAIR I had issue with the tests. Maybe it works in the free wild without the script reference. Interesting question in this context: we do we have `writetoDisk(./.)` with the script in the signature? Do we need that here?